agent/src/IngestDetailsRepository.php

```diff
@@ -39,11 +39,13 @@ class IngestDetailsRepository
     private ?array $quickRetryStrategyDurationsCache = null;
 
     /**
+     * @param  LoopInterface  $loop
      * @param  Browser  $browser
      * @param  (Closure(IngestDetails $ingestDetails, float $duration): mixed)  $onAuthenticationSuccess
      * @param  (Closure(Throwable $e, float $duration): mixed)  $onAuthenticationError
      */
     public function __construct(
+        private $loop,
```

Comment:
To allow testing, I'm not adding a type declaration to this in the same way the `$browser` does not have a type declaration.